asf_heat_pump_suitability/pipeline/transform/heat_network_zones.py

```diff
@@ -0,0 +1,42 @@
+"""
+Functions to filter OS UPRNs to UPRNs located within heat network zones.
```

Comment:
```suggestion
Functions to filter UPRNs to those located within heat network zones.
```
To clarify that this function can be used for UPRNs of any origin.